test/accreditation/generate_data.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+# pylint: disable= no-member
+"""
+Generate data for accreditation tests
+"""
+
+# Import general libraries (needed for functions)
+import pickle
+import qiskit
+
+
+# Import Qiskit classes
+from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister, execute
+from qiskit.providers.aer.noise import NoiseModel
+from qiskit.providers.aer.noise.errors.standard_errors import depolarizing_error
+
+# Import the accreditation functions.
+from qiskit.ignis.verification.accreditation import AccreditationFitter, AccreditationCircuits
+
+
+def make_accred_system():
+    """generate accred circuits"""
+    # Create a Quantum Register with n_qb qubits.
+    q_reg = QuantumRegister(4, 'q')
+    # Create a Classical Register with n_qb bits.
+    c_reg = ClassicalRegister(4, 's')
+    # Create a Quantum Circuit acting on the q register
+    target_circuit = QuantumCircuit(q_reg, c_reg)
+    target_circuit.h(0)
+    target_circuit.h(1)
+    target_circuit.h(2)
+    target_circuit.h(3)
+    target_circuit.cx(0, 1)
+    target_circuit.cx(0, 2)
+    target_circuit.h(1)
+    target_circuit.h(2)
+    target_circuit.cx(0, 3)
+    target_circuit.cx(1, 2)
+    target_circuit.h(1)
+    target_circuit.h(2)
+    target_circuit.h(3)
+    target_circuit.measure(q_reg, c_reg)
+
+    return AccreditationCircuits(target_circuit)
+
+
+def generate_data_ideal():
+    """generaete a ideal data pickle"""
+    accsys = make_accred_system()
```

Comment:
This could be updated to have a fixed simulator seed and accreditation seed. Eg
```python
seed_accreditation = 1010
seed_simulator = 2020

accsys = make_accred_system(seed=seed_accreditation)
```